tests/ci_visibility/test_encoder.py

```diff
@@ -199,6 +199,10 @@ def subprocess_run(self, *args):
         """Execute test script with test tracer."""
         return self.testdir.runpytest_subprocess(*args)
 
+    def teardown(self):
+        if CIVisibility.enabled:
```

Comment:
would it make sense to make this an auto fixture?

Reply:
Will try this in the future, for now the teardown is enough